### DISCUSSION TITLE
How do I update Windows Subsystem For Android

### DISCUSSION THREAD

**Discussion by OpokuKoduah** [LSPosed/MagiskOnWSALocal#391] [Q&A]
I want to update windows Windows Subsystem For Android without reinstalling a new one or having to update it via Windows Updates?

**Comment by icefrog119** [comment]
first backup  all files you got from the version you got installed on your computer now on a cloud drive,  USB drive  whatever    then delete the old files  from the old version  that's on your computer   then  build your Windows Subsystem for Android locally on your computer  like you did to install the version you got installed now    then go to the new version files  that you created locally   right click the install file  click open with powershell   then it will update windows subsystem for android
